Dockerfile.shipper-state-metrics

```diff
@@ -1,6 +1,5 @@
-ARG BASE_IMAGE=alpine:3.8
```

Comment:
This is used by internal CI.

Reply:
Given that we want to use the same Dockerfile in both environments, I think we should be more conservative with Docker features (for example, using microk8s and Docker in the same host is discouraged, which enables this feature due to microk8s current version is 17.03 and the ARG before FROM feature appeared in 17.05).